src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProjectTracker.cs

```diff
@@ -626,26 +623,68 @@ private void InitializeOutputPane()
             cancellationToken.ThrowIfCancellationRequested();
             OutputToOutputWindow($"Getting project information - done (took {DateTimeOffset.UtcNow - start})");
 
+            ForceLoadProjectsWhoseDesignTimeBuildFailed(projectInfos);
```

Comment:
Not without passing `VSBSLFLAGS_LoadBuildDependencies` to `IVsSolution4.EnsureProjectsAreLoaded`.

Also `CreateDeferredProjects` should handle that fine - if the project is "real" loaded, it will no-op.  This is also necessary for:

1. Projects that don't support deferred loading.
2. Projects that the user might have triggered loading before `CreateDeferredProjects` is called (either explicitly by expanding them, or because they were expanded in the .suo file).